src/Components/ProjectForm.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { ReactSession } from 'react-client-session';
+import styled from "styled-components";
+import TextField from '@mui/material/TextField';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/CSC510-Group-25/feature-hunt/pullRequest?prid=8358184)
```suggestion
import TextField from "@mui/material/TextField";
```